gocd/gocd.go

```diff
@@ -101,12 +102,19 @@ func client() *http.Client {
 }
 
 // NewServerConfig Create a Server object from a config
-func NewServerConfig(host string, port string, user string, password string) *Server {
+func NewServerConfig(host string, port string, user string, password string, timeoutStr string) *Server {
+	timeout := time.Duration(120 * time.Second)
```

Comment:
Is this meant to be a default in case an error occurs in the string conversion?  If so, I'd say it's better to put this in an else block, and log a warning.